### PR TITLE
Add entites transfer manager

### PR DESCRIPTION
**Pull request by leonprou**

### CODE REVIEW

File: test/entitiesTransferManager.js [hunk 1]
```diff
@@ -0,0 +1,301 @@
+// const ERC677BridgeToken = artifacts.require('ERC677BridgeToken.sol')
```
Comment (LiorRabin) [inline test/entitiesTransferManager.js]: remove comment

File: test/entitiesTransferManager.js [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+    beforeEach(async () => {
+      await transferManager.addEntity(user, entity.uri, entity.permissions, { from: owner }).should.be.fulfilled
+    })
+    it('owner can remove entity', async () => {
+      await transferManager.removeEntity(user, { from: owner }).should.be.fulfilled
+      await validateNoEntity(user)
+    })
+
+    it('only owner can add remove entity', async () => {
```
Comment (LiorRabin) [inline test/entitiesTransferManager.js]: remove "add"

File: test/mockContracts/EntitiesTransferManagerMock.sol [hunk 3]
```diff
@@ -0,0 +1,12 @@
+pragma solidity 0.4.24;
```
Comment (LiorRabin) [inline test/mockContracts/EntitiesTransferManagerMock.sol]: why you need the mock contract? isn't it the same functionality as the `EntitiesTransferManager` itself?

Comment (leonprou) [reply test/mockContracts/EntitiesTransferManagerMock.sol]: right! I needed it  to tess fucntion overloading (two `addRule` fucntions in the contract). Now I can use function overloading in truffle v5 without the mock contract.

**Pull request merged**